sdk/cosmos/examples/pom.xml

```diff
@@ -95,64 +83,65 @@
           </execution>
         </executions>
       </plugin>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-antrun-plugin</artifactId>
-        <version>1.8</version>
-        <inherited>false</inherited>
-        <executions>
-          <execution>
-            <phase>none</phase>
-            <id>default-cli</id>
-          </execution>
-        </executions>
-        <configuration>
-          <skip>true</skip>
-        </configuration>
-      </plugin>
+
+      <!--<plugin>-->
+        <!--<groupId>org.apache.maven.plugins</groupId>-->
+        <!--<artifactId>maven-antrun-plugin</artifactId>-->
+        <!--<inherited>false</inherited>-->
+        <!--<executions>-->
+          <!--<execution>-->
+            <!--<phase>none</phase>-->
+            <!--<id>default-cli</id>-->
+          <!--</execution>-->
+        <!--</executions>-->
+        <!--<configuration>-->
+          <!--<skip>true</skip>-->
+        <!--</configuration>-->
+      <!--</plugin>-->
```

Comment:
the intention of the ant plugin was to collect the artifacts, renaming for publish.

as now we rely on your process for release, this can be removed.

Reply:
removed.